mmv1/third_party/terraform/website/docs/d/compute_reservation.html.markdown

```diff
@@ -0,0 +1,44 @@
+---
+subcategory: "Compute Engine"
+description: |-
+  Provide access to a Reservation's attributes
+---
+
+# google\_compute\_reservation
+
+Provides access to available Google Compute Reservation Resources for a given project.
+See more about [Reservations of Compute Engine resources](https://cloud.google.com/compute/docs/instances/reservations-overview) in the upstream docs.
+
+```hcl
+data "google_compute_reservation" "reservation" {
+  name = "gce-reservation"
+  zone = "us-central1-a"
+}
+
+resource "google_compute_reservation" "test" {
+  name = "gce-reservation"
+  zone = "us-central1-a"
+  project = "PROJECT_ID"
+
+
+  specific_reservation {
+    count = 3
+    instance_properties {
+      min_cpu_platform = "Intel Cascade Lake"
+      machine_type     = "n2-standard-2"
+    }
+  }
+}
```

Comment:
Generally speaking you only need to include another resource here if it's _using_ the data source.

```suggestion
```